### PR TITLE
add configuration override to prompty example

### PR DESCRIPTION
**Pull request by lalala123123**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: examples/prompty/basic/prompty-quickstart.ipynb [hunk 1]
```diff
@@ -79,10 +79,79 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from promptflow.core import Flow\n",
+    "from promptflow.core import Prompty, AzureOpenAIModelConfiguration, OpenAIModelConfiguration\n",
     "\n",
     "# load prompty as a flow\n",
-    "f = Flow.load(\"basic.prompty\")\n",
+    "f = Prompty.load(source=\"basic.prompty\")\n",
```
Comment (wangchao1230) [inline examples/prompty/basic/prompty-quickstart.ipynb:85]: not make them commented so our CI can guard this?

File: examples/prompty/basic/prompty-quickstart.ipynb [hunk 2]
```diff
@@ -79,10 +79,79 @@
...
+    "# override_model = {\n",
+    "#     \"configuration\": configuration,\n",
+    "#     \"parameters\": {\n",
+    "#         \"max_token\": 512\n",
+    "#     }\n",
+    "# }\n",
+    "# f = Prompty.load(source=\"basic.prompty\", model=override_model)\n",
+    "\n",
+    "# override configuration with created connection in AzureOpenAIModelConfiguration\n",
+    "# configuration = AzureOpenAIModelConfiguration(\n",
```
Comment (wangchao1230) [inline examples/prompty/basic/prompty-quickstart.ipynb:116]: put the conneciton one into the chat notebook which is using connection?

Comment (lalala123123) [reply examples/prompty/basic/prompty-quickstart.ipynb]: move connection into chat notebook

File: examples/prompty/basic/prompty-quickstart.ipynb [hunk 3]
```diff
@@ -79,78 +79,105 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
     "# override model configuration of prompty with dict\n",
+    "override_model = {\n",
+    "    \"configuration\": {\n",
+    "        \"type\": \"azure_openai\",\n",
+    "        \"azure_deployment\": \"gpt-35-turbo\",\n",
+    "        \"api_key\": \"${env:AZURE_OPENAI_API_KEY}\",\n",
```
Comment (wangchao1230) [inline examples/prompty/basic/prompty-quickstart.ipynb:112]: add comment say user can point api_key to other env variable

Comment (lalala123123) [reply examples/prompty/basic/prompty-quickstart.ipynb]: fix it

**Pull request merged**